flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -744,7 +745,8 @@ class ExprLowering {
               mlir::Value total = zero;
               assert(arr.shape.size() == aref.subscript().size());
               for (const auto &pair : llvm::zip(arr.shape, aref.subscript())) {
-                auto val = builder.create<fir::ConvertOp>(loc, idxTy,genval(std::get<1>(pair)));
+                auto val = builder.create<fir::ConvertOp>(
+                    loc, idxTy, genval(std::get<1>(pair)));
```

Comment:
BTW, what version of clang-format are you using?

I'm using one the one built from llvm-project:HEAD a couple of days ago, and it did not reindent any of this code. <sigh>

Reply:
I'm happy to use any formatting version that does the job.  An attempt to use 'git clang-format' does look like it could be a little annoying though:

git help clang-format
No manual entry for git-clang-format

The following files would be modified but have unstaged changes:
M       flang/lib/Lower/ConvertExpr.cpp
Please commit, stage, or stash them first.

Reply:
It's weird. <sigh>